ts/components/messages/MessageDetailComponent.tsx

```diff
@@ -212,8 +212,6 @@ export default class MessageDetailComponent extends React.PureComponent<
             </React.Fragment>
           )}
         </Content>
-        <View spacer={true} large={true} />
-        <View spacer={true} small={true} />
```

Comment:
This change leads to a problem at least on IPhone 11.
<img src="https://user-images.githubusercontent.com/11773070/107505486-0c941180-6b9d-11eb-8639-5b8b4d408e5a.png" width=300>

Reply:
![Screenshot 2021-02-19 at 20 55 44](https://user-images.githubusercontent.com/8376065/108554925-116f6880-72f5-11eb-8cc0-9eb511a7c798.png)